rarbgapi/rarbgapi.py

```diff
@@ -127,6 +127,8 @@ def wrapper(self, *args, **kwargs):
                 resp = func(self, token=self._token, *args, **kwargs)
                 json_ = resp.json(object_hook=json_hook)
                 return json_['torrent_results']
+            except KeyError:
```

Comment:
No, I get:

    Unexpected exceptin 'torrent_results'
    Traceback (most recent call last):
      File "/Users/easyb/Documents/src/rarbgapi/rarbgapi/rarbgapi.py", line 130, in wrapper
        return json_['torrent_results']
    KeyError: 'torrent_results'